_sections/30-bindings.md

```diff
@@ -145,24 +145,59 @@ A subset of [XPath 1.0 functions](http://www.w3.org/TR/xpath/#corelib), some fun
 | `area(node-set ns | geoshape gs)`			| Returns the calculated area in m2 of either a nodeset of geopoints or a geoshape value (not a combination of both) on Earth. It takes into account the circumference of the Earth around the Equator but does not take altitude into account.	
 | `once(* calc)`						| The parameter will be evaluated and returned if the context nodes's value is empty, otherwise the current value of the context node will be returned. The function is used e.g. to ensure that a random number is only generated once with `once(random())`.
 
-### Preloaders - Metadata
+### Metadata
 
-To be discussed. Not complete and not correct.
-
-[review]()
-
-| jr:preload    | jr:preloadParams | node            | description
-|---------------|------------------|-----------------|-------------------
-| instance      |                  | /meta/instanceID| Unique Instance ID generated by concatenating 'uuid:' with the value generated by the uuid() XPath function.
-| timestamp     | start            | /meta/timeStart | Timestamp in datetime data format when the user opened the form. Only populated once.
-| timestamp     | end              | /meta/timeEnd   | Timestamp in datetime data format when the user last saved the form. 
-| property   	| deviceid         |      tbc        | Unique identifier of device. Guaranteed not to be blank but could be 'not supported'. Either the cellular IMEI (with imei: prefix, e.g. imei:A0006F5E212), WiFi mac address (with mac: prefix, e.g mac:01:23:45:67:89:ab), Android ID (e.g. android_id:12011110), or another unique device ID for a webbased client (with domain prefix,e .g. enketo.org:SOMEID).
-| property		| email            |      tbc        | Populate with the user's email address if the client has access to this, otherwise 'not supported'.
-| property 		| username         |      tbc        | Populate with username stored in the client. Can be blank if not set or 'not supported'. 
-| property      | phone number     |      tbc        | Populate with phone number of device. Can be blank if not available (e.g. for desktop or tablet) or 'not supported'.
-| property      | simserial        |      tbc        | SIM serial number of phone. May be blank (e.g. for desktop or tablet or 'not supported').
-| property      | subscriberid	   |      tbc        | IMSI of phone prefixed (with imsi: prefix, e.g. imsi:SD655E212). May be blank (e.g., tablets).
+This section describes metadata about _the record_ that is created with the form. Metadata about _the form itself_ (id, version, etc) is covered in the [Primary Instance](http://localhost:4000/#primary-instance) section.
 
+The namespace of the meta block is either the default XForms namespace or "https://openrosa.org/xforms".
```

Comment:
Yes, I think so. Then tools can decide whether they want to be fully compliant or more lenient.

But I do want to verify my understanding of something. I know JavaRosa isn't something you're particularly interested in but perhaps you have a general explanation for what's going on -- JR doesn't have any knowledge of the openrosa.org/xforms namespace. So is it just that it currently accepts any namespace for the meta block? I suppose this is also what CommCare does? If you're not sure I can also dig through the code.